src/sota_tools/deploy.h

```diff
@@ -43,4 +43,10 @@ bool OfflineSignRepo(const ServerCredentials& push_credentials, const std::strin
 bool PushRootRef(const ServerCredentials& push_credentials, const OSTreeRef& ref, const std::string& cacerts,
                  RunMode mode);
 
+/**
```

Comment:
Ok that works but it's maybe a bit confusing to have this function as part of `deploy.cc`. Maybe just add it to a new translation unit `check.cc` and `check.h`.